users/models.py

```diff
@@ -45,3 +45,20 @@ def delete(self, *args, **kwargs):
 
     def __str__(self) -> str:
         return self.user.username
+
+    def get_user_role(self):
+        return self.role
+
+    @staticmethod
+    def is_employee(user):
+        if user.get_user_role() == Role.EMPLOYEE:
+            return True
+        else:
+            return False
```

Comment:
Why using staticmethod? and you can return the boolean itself:
```suggestion
    def is_employee(self):
        return self.role == Role.EMPLOYEE:
```